src/components/shop/backgrounds.js

```diff
@@ -41,7 +41,7 @@ export const initResizers = () => {
     resizers.set(Object, () => {});
 
     resizers.set(Phaser.GameObjects.Image, (scene, background, newSpec = {}) => {
-        const spec = { ...newSpec, ...defaultSpec };
+        const spec = { ...defaultSpec, ...newSpec };
```

Comment:
only code change, seems to have no impact 🤞

Reply:
surprised it didn't cause issues the other way around